src/app/api/[[...route]]/routes/tenant.route.ts

```diff
@@ -229,6 +234,25 @@ const app = new Hono()
             404,
           );
 
+        const tenants = RotationAssignment.tenantPlaceholders.map(
+          (tenantPlaceholder) => {
+            return tenantPlaceholder.tenant;
+          },
+        );
+
+        if (tenants.length >= CONSTRAINTS.TENANT_MAX_AMOUNT) {
+          return c.json(
+            {
+              error: SERVER_ERROR_MESSAGES.MAX_LIMIT_REACHED(
+                'tenants',
+                CONSTRAINTS.TENANT_MAX_AMOUNT,
+              ),
+            },
+            400,
+          );
+        }
+        console.log('tenant', tenants);
```

Comment:
Could you remove this console.log please?

Reply:
I removed that.